src/index.js

```diff
@@ -40,106 +40,122 @@ window.addTask = function addTask() {
       }
       return 0;
     });
-
-    window.updateLocalStorage(false);
+    window.update(tasks);
   }
 };
 
-window.editTask = function editTask(data) {
-  const list = document.getElementsByTagName('li');
-
-  Array.from(list).forEach((li) => {
-    if (li.id === data.id) {
-      li.style.backgroundColor = '#fff59c78';
-      const img = li.getElementsByTagName('img')[0];
-
-      img.src = TrashImg;
-    } else {
-      li.style.backgroundColor = 'white';
-      const img = li.getElementsByTagName('img')[0];
-      img.src = MoreImg;
+window.updateLocalStorage = function updateLocalStorage(retrieve) {
+  if (retrieve === true) {
+    if (tasks === null) {
+      tasks = JSON.parse(window.localStorage.getItem('tasks'));
     }
-  });
-};
-
-window.removeTask = function removetask(data) {
-  let id;
-  if (!data.id) {
-    id = data;
   } else {
-    id = data.id;
+    window.localStorage.setItem('tasks', JSON.stringify(tasks));
   }
-  const temp = [];
-  tasks.forEach((task) => {
-    if (task.id !== id) {
-      temp.push(task);
-    }
-  });
-  tasks = temp;
-  window.updateLocalStorage(true);
+  window.displayTasks();
 };
 
-window.clear = function clear() {
-  const temp = [];
-  tasks.forEach((task) => {
-    if (task.completed !== true) {
-      temp.push(task);
-    }
-  });
-  tasks = temp;
-  window.updateLocalStorage(true);
+window.callAddTask = function callAddTask() {
+  window.addTask(tasks);
 };
-window.markcompleted = function markcompleted(id) {
-  tasks.find((task) => task.id === id).completed = true;
+
+window.restart = function restart() {
+  tasks = null;
+  window.updateLocalStorage(false);
 };
 
-window.updateLocalStorage = function updateLocalStorage(remove) {
-  if (remove !== true) {
-    if (tasks === null) {
-      tasks = JSON.parse(window.localStorage.getItem('tasks'));
-    }
+window.update = function update(data) {
+  if (!data) {
+    const response = updateTasks();
+    tasks = response;
+  } else {
+    tasks = data;
   }
 
-  window.localStorage.setItem('tasks', JSON.stringify(tasks));
-  window.displayTasks();
+  window.updateLocalStorage(false);
 };
```

Comment:
- [ ] The updated array of items  is supposed to be stored in local storage, so the user gets the correct list values after the page reloads. Please work on that!